### PR TITLE
Adds some additional banners

### LINKED ISSUE #11394: "npm run add add-simulated-synopsis-visits" does not work on master
## Test plan

https://github.com/brave/browser-laptop/pull/12014#issue-274793402

----

### Description

`npm run add-simulated-synopsis-visits` does not work on master.

````
Uncaught Exception:
Error: ENOENT: no such file or directory, open '/Users/Suguru/Library/Application Support/brave-development/ledger-synopsis.json'
    at Object.fs.openSync (fs.js:584:18)
    at Object.module.(anonymous function) [as openSync] (ELECTRON_ASAR.js:168:20)
    at Object.fs.readFileSync (fs.js:491:33)
    at Object.fs.readFileSync (ELECTRON_ASAR.js:501:29)
    at updateExistingSynopsisFile (/Users/Suguru/browser-laptop/tools/lib/synopsisHelpers.js:26:45)
    at addSimulatedSynopsisVisits (/Users/Suguru/browser-laptop/tools/lib/utilApp/index.js:47:3)
    at App.app.on (/Users/Suguru/browser-laptop/tools/lib/utilApp/index.js:60:7)
    at emitTwo (events.js:111:20)
    at App.emit (events.js:194:7)
````

### Steps to Reproduce

  1. `npm run add-simulated-synopsis-visits`

**Actual result:**

See above.

**Expected result:**

**Reproduces how often:** [What percentage of the time does it reproduce?]

### Brave Version

**about:brave info:**

433e69fb0efc0be3ed3b5370e4b46f40612aa686

**Reproducible on current live release:**

### LINKED ISSUE #12098: Claim token button should be hidden if a wallet is recovered with ugp token
### Description
Claim token button should be hidden if a wallet is recovered with ugp token

### Steps to Reproduce 
   1. Clean install 0.19.96
   2. Create wallet and claim ugp tokens
   3. Backup wallet and clear browser profile
   4. Create a new browser profile
   5. Enable wallet and recover the wallet from step 3 
   6. Claim free token button is still shown, clicking on the button shows promotion not available

**Actual result:**
```
>>> {"statusCode":422,"error":"Unprocessable Entity","message":"promotion already in use"}
Problem claiming promotion Error: HTTP response 422 for PUT /v1/grants/51245323-b45b---ee0584e183de 
```

**Expected result:**
If wallet already contains ugp tokens, claim token button should not be shown

**Reproduces how often:**
100%

### Brave Version

**about:brave info:**

Brave | 0.19.96
-- | --
rev | 9d72944
Muon | 4.5.16
libchromiumcontent | 62.0.3202.94
V8 | 6.2.414.42
Node.js | 7.9.0
Update Channel | Release
OS Platform | Microsoft Windows
OS Release | 10.0.15063
OS Architecture | x64

**Reproducible on current live release:**
N/A

### Additional Information
Confirmed by @LaurenWags on macOS

### LINKED ISSUE #12131: Add UGP banners
### Description

Add additional banners when there is an error or promotion expired.

(copied from https://github.com/brave/browser-laptop/issues/11100#issuecomment-347731485)
@NejcZdovc Here is the language for the 3 message panels for UGP:

image

Congrats! 25 Tokens have been added to your account!

Please note that these tokens will expire in 90 days if not used, in whole or in part, towards your monthly Brave Payments contribution.

________
Sorry! The promotion has ended.

There may be additional Basic Attention Token promotions in the future so stay tuned!

_________

Uh oh. The Brave Payments server is not responding. Please try again later to claim your token grant.

Note: This error could also be caused by a network connection problem.

### PR DESCRIPTION
**Pull request by NejcZdovc**
# This PR switches promotions to the production server!!

Please test this PR with `LEDGER_ENVIRONMENT=staging npm run start`, so that you will still be using staging.

Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).

Fixed recovery promotional removal
Removes staging flag
Fixes fake visits

Resolves #12131
Resolves #12098
Resolves #11394

Auditors:

Test Plan:

a) Regular promotion
1) enable wallet
2) wait for promotion to appear
3) click on claim
4) make sure that success message appears

b) Network problem
1) enable wallet
2) wait for promotion to appear
3) disable internet
3) click on claim
4) make sure that general error message appears

Reviewer Checklist:

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

**Comment by srirambv** [comment]
`The rest is easy` doesn't have a line break before it. Is this expected?
image

**Comment by srirambv** [comment]
Getting error  `Error retrieving promotion SyntaxError: Unexpected token # in JSON at position 0`  while launching the browser

**Comment by LaurenWags** [comment]
Looks good on MacOS

**Comment by kjozwiak** [comment]
> Resolves #12131
> Resolves #12098
> Resolves #11394

Checked these against Ubuntu... Everything worked as expected... Will test more thoroughly once this PR has been merged.

File: js/actions/appActions.js [hunk 1]
```diff
@@ -1837,9 +1837,10 @@ const appActions = {
     })
   },
 
-  onPromotionResponse: function () {
+  onPromotionResponse: function (error) {
     dispatch({
-      actionType: appConstants.APP_ON_PROMOTION_RESPONSE
+      actionType: appConstants.APP_ON_PROMOTION_RESPONSE,
+      error
```
Comment (bsclifton) [inline js/actions/appActions.js]: Are we wanting to call this error? It seems like this also contains the success response?

Comment (NejcZdovc) [reply js/actions/appActions.js]: renamed

**Comment by bsclifton** [comment]
master 81f7991ed8b2cfd4a411fee2b638cfd60f090db0
0.21.x 03e093ec729b146a38fcf257e4cfffcdd9ba91cd
0.20.x 204a094ff6aa9e9a7216a383249db1971c6fdde2
0.19.x @NejcZdovc to help with this one...

**Comment by NejcZdovc** [comment]
0.19.x 23d2f5a06

**Pull request merged**